src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age ? <p className="Person__age">{`I am ${person.age}`}</p> : ''}
+    <p className="Person__partner">
+      {/* eslint-disable-next-line no-nested-ternary */}
+      {person.isMarried
+        ? person.sex === 'm'
+          ? `${person.partnerName} is my wife`
+          : `${person.partnerName} is my husband`
```

Comment:
move this ternary to separate func, or refactor it a bit for avoid nested ternary operations